src/actions/bitcoinActions.js

```diff
@@ -54,10 +55,10 @@ import { saveDbAction } from 'actions/dbActions';
 const storage = Storage.getInstance('db');
 
 const saveDb = (data: BitcoinStore) => {
-  return saveDbAction('bitcoin', data, true); // TODO: +true+ required?
+  return saveDbAction('bitcoin', data);
```

Comment:
`true` flag means you want to override the value instead of merging. Merging could do you a bad job when you merge arrays and when you want to remove some options from the objects

Reply:
Alright, added it back.